src/go/rpk/pkg/cli/cmd/redpanda/config.go

```diff
@@ -246,6 +253,12 @@ func parseSeedIPs(ips []string) ([]config.SeedServer, error) {
 		}
 		seeds = append(seeds, seed)
 	}
+	if len(seeds) == 0 {
+		return nil, errors.New("Must provide seed servers in --ips")
+	}
+	if len(seeds) < 3 {
+		fmt.Println("It is strongly recommended that --ips refers to at least three nodes")
```

Comment:
Not sure how I feel about this -- this would be the first "we suggest something but also we'll continue".

Reply:
Moved this into the flag description